lib/backend/pgbk/pgbk_test.go

```diff
@@ -0,0 +1,69 @@
+// Copyright 2023 Gravitational, Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package pgbk
+
+import (
+	"context"
+	"encoding/json"
+	"os"
+	"testing"
+
+	"github.com/gravitational/trace"
+	"github.com/jonboulle/clockwork"
+	"github.com/stretchr/testify/require"
+
+	"github.com/gravitational/teleport/lib/backend"
+	"github.com/gravitational/teleport/lib/backend/test"
+	"github.com/gravitational/teleport/lib/utils"
+)
+
+func TestMain(m *testing.M) {
+	utils.InitLoggerForTests()
+	os.Exit(m.Run())
+}
+
+func TestPostgresBackend(t *testing.T) {
+	// recommended: "expiry_interval":"500ms","change_feed_poll_interval":"500ms"
+	paramString := os.Getenv("TELEPORT_PGBK_TEST_PARAMS_JSON")
```

Comment:
Where is the connection string? In other words, how will this test know where the instance is?

Reply:
Updated comment in e364d78fa710cc9fc82e2755d5bf52689d3dbdc1: those are options that should be added, you are meant to pass a json blob containing all the params including `"conn_string":"..."`.